resources/views/admin/users/index.blade.php

```diff
@@ -196,6 +228,17 @@
                   }).catch(error => {
                     this.addError = error.response.data.errors;
                     this.disabled = false;
+                    if (!Object.values(this.addError).some(field => field.includes('userExists'))) {
+                        if (this.addError.email) {
+                            restoreUserModal.emailToRestore = restoreUserModal.restoreValue = this.email;
+                            restoreUserModal.restoreAttribute = 'email address';
+                        } else if (this.addError.username) {
+                            restoreUserModal.usernameToRestore = restoreUserModal.restoreValue= this.username;
+                            restoreUserModal.restoreAttribute = 'username';
+                        }
+                        $('#addUser').modal('hide');
```

Comment:
Is this jQuery?!? Please update to use VueJS https://bootstrap-vue.js.org/docs/components/modal/